src/main/java/joe/parser/Parser.java

```diff
@@ -0,0 +1,158 @@
+package joe.parser;
+
+import joe.exception.InvalidJoeInputException;
+import joe.storage.Storage;
+import joe.task.Deadline;
+import joe.task.Event;
+import joe.task.TaskList;
+import joe.task.ToDo;
+import joe.task.Task;
```

Comment:
```suggestion
import joe.task.Task;
import joe.task.TaskList;
import joe.task.ToDo;
```
Nit: Sort the import ordering to make it easier to navigate?